Rakefile

```
@@ -7,7 +7,26 @@ rescue LoadError
   abort 'Please run this task using `bundle exec rake`'
 end
 
+task default: %i[test rubocop]
+
 RuboCop::RakeTask.new
-task default: %i[spec rubocop]
+
+desc 'Run the unit tests using rspec'
+task :test do
+  sh('rspec')
+end
+
+desc "Generate the docs using YARD"
+task :doc do
+  sh 'yard', 'doc'
```

Nipick, since all the other `sh` calls use parenthesis:

```suggestion
  sh('yard', 'doc')
```